### PR TITLE
[ZH] Update `Medals`

### PR DESCRIPTION
**Pull request by cdwcgt**
Co-Authored-By: Dead_Bush_Sanpai <46657591+DeadBushSanpai@users.noreply.github.com>

- [x] Describe of `Hush-Hush`
- [x] Need to check `Beatmap Challenge Packs`
- [x] Spotlight

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/zh.md [hunk 1]
```diff
@@ -1,77 +1,59 @@
----
-outdated: true
----
-
 # 奖章
 
-奖章，曾被称为 *成就*，是在用户个人资料页面中展示出来的一枚枚小型图案，可以体现玩家的游戏水准和技巧。
-当前有5种奖章：[曲包](#曲包)，[实力](#实力)，[毅力](#毅力)，[Mod 初体验](#Mod%20初体验)，和[隐藏](#隐藏)。
+**奖章**，曾被称为 *成就*，是在用户个人资料页面中展示出来的一枚枚小型图案，可以体现玩家的游戏水准和技巧。目前，在如下八个分类下有共计**247**枚奖章：[曲包](#曲包)，[实力](#实力)，[毅力](#毅力)，[Mod 入门](#Mod-入门)，[隐藏](#隐藏)，[谱面聚光灯](#beatmap-spotlights)，[季节聚光灯](#seasonal-spotlights)，和[谱面挑战曲包](#谱面挑战曲包)。
```
Comment (Kotoki1337) [inline wiki/Medals/zh.md:3]:
```suggestion
**奖章**，曾被称为 *成就*，是在用户个人资料页面中展示出来的一枚枚小型图案，可以体现玩家的游戏水准和技巧。目前，在如下八个分类下有共计**247**枚奖章：[曲包](#曲包)，[实力](#实力)，[毅力](#毅力)，[Mod 入门](#Mod-入门)，[隐藏](#隐藏)，[聚光灯谱面](#beatmap-spotlights)，[聚光灯季节](#seasonal-spotlights)，和[挑战谱面曲包](#挑战谱面曲包)。
```

我觉得这样听起来更顺一点，括号里的跳转记得改成和下文标题相同。

File: wiki/Medals/zh.md [hunk 2]
```diff
@@ -1,77 +1,59 @@
...
-实力奖章目前有三类：连击数，排名，水准。
+## Mod 入门
 
-**连击数**奖章获取条件：在 osu! 模式中玩任意谱面并取得指定[连击数](/wiki/Disambiguation/Combo)。
-从 [osu!taiko](/wiki/Game_mode/osu!taiko)，[osu!catch](/wiki/Game_mode/osu!catch) 或 [osu!mania](/wiki/Game_mode/osu!mania) 模式中取得的连击数不计入奖章获取条件。
+*参见：[Mods](/wiki/Game_modifier)。*
 
-如果想要取得 1000 或 2000 连击数的奖牌，通常您需要搜索 Approved 谱面，而不是 Ranked 谱面。因为 Approved 谱面的谱面时间将会更长（5分钟以上），这样更容易取得更高的连击数。
-注意，如果谱面既没有被 Approved，也没有被 Ranked，那么即使连击数再高，也不会取得奖章！
+**mod 入门**是在特定的[游戏模组（mod）](/wiki/Game_modifier)下通过[Qualified](/wiki/Beatmap/Category#qualified)，[Ranked](/wiki/Beatmap/Category#ranked)，或者 [Approved](/wiki/Beatmap/Category#approved)谱面获得。这些奖章是为了激励新玩家尝试并体验各种各样的 mod 而设立的。但是请注意，如果玩家同时使用多个 mods。则玩家**不会**获得*任何* mod 奖章。
```
Comment (Kotoki1337) [inline wiki/Medals/zh.md:25]:
```suggestion
**Mod 入门**是在特定的[游戏模组（Mod）](/wiki/Game_modifier)下通过 [Qualified](/wiki/Beatmap/Category#qualified)，[Ranked](/wiki/Beatmap/Category#ranked)，或者 [Approved](/wiki/Beatmap/Category#approved) 谱面获得。这些奖章是为了激励新玩家尝试并体验各种各样的 Mod 而设立的。但是请注意，如果玩家同时使用多个 Mods。则玩家**不会**获得*任何* Mod 奖章。
```

format i think?

File: wiki/Medals/zh.md [hunk 3]
```diff
@@ -1,77 +1,59 @@
...
 
----
+除非另有说明，否则这些奖章都可以在任何 [游戏模式](/wiki/Game_mode) 中解锁。
 
-排名奖章的获取方法是：在**任意**游戏模式中取得一定全球排名。
+## 隐藏
 
----
+**隐藏奖章**通过*神秘*的方式来获得，不会具体告诉玩家获得奖章的方法(因此得名)。使用特殊方法通过谱面。但是仍有一些提示给那些想要收集全奖章的人，如下所示：
```
Comment (Kotoki1337) [inline wiki/Medals/zh.md:31]:
```suggestion
**隐藏奖章**通过*神秘*的方式来获得，不会具体告诉玩家获得奖章的方法（因此得名）。使用特殊方法通过谱面。但是仍有一些提示给那些想要收集全奖章的人，如下所示：
```
another format

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]

File: wiki/Medals/zh.md (hunk 1 above)
Comment (cdwcgt) [reply wiki/Medals/zh.md:3]: 聚光灯季节是不是有点太怪了
我到可能更偏向于 `季节性` 的修饰而不是把他当作一个 `季节` 的名字
挑战同理 还不如直接叫挑战曲包

聚光灯谱面倒是觉得可以改

Comment (Kotoki1337) [reply wiki/Medals/zh.md:3]: 我读了八百万次感觉聚光灯放后面太怪了，怎么读都读不顺。这个 Seasonal 确实有点不好处理，然后根据原文的 `Although the Seasonal Spotlights have been discontinued as of early 2020 in favour of the current Beatmap Spotlights system`，还需要跟新系统赛季做出对应区分，也许我们可以更进一步，使用 `季节性聚光灯曲包`，又可以和下面的 `挑战曲包`相适应，也从标题引出了该奖章需要完成曲包内谱面获得。What do you think?

File: wiki/Medals/zh.md (hunk 3 above)
Comment (cdwcgt) [reply wiki/Medals/zh.md]: fix at 4b1db8cc4229d7c3be8f9266d14ea628e58d2400

File: wiki/Medals/zh.md (hunk 2 above)
Comment (cdwcgt) [reply wiki/Medals/zh.md]: fix at 4b1db8cc4229d7c3be8f9266d14ea628e58d2400

File: wiki/Medals/zh.md [hunk 4]
```diff
@@ -1,77 +1,59 @@
...
 ## 曲包
 
-**曲包**奖章的获得方法是： **完整体验特定曲包中的所有谱面** （每张至少选任意一难度）。您必须在 [osu!](/wiki/Game_mode/osu!) 模式中 pass 曲包中的每一张图，才能得到对应的奖章。
+**曲包奖章**在完成某些[主题曲包](https://osu.ppy.sh/beatmaps/packs?type=theme)或者[艺术家/专辑曲包](https://osu.ppy.sh/beatmaps/packs?type=artist)后获得。这些奖章通过完成曲包中每个谱面集的其中一个难度来获得。
 
-当前共有16个曲包奖章，分为4大类：电子游戏，动漫，网络，以及音乐游戏，其中每一大类有4个曲包。[查看并下载主题曲包](https://osu.ppy.sh/p/packlist?t=t)
+除非另外说明，否则必须在 [osu!](/wiki/Game_mode/osu!) 游戏模式中完成谱面才有资格获得奖章。允许使用[降低难度的 mods](/wiki/Game_modifier#difficulty-reduction) 和[提升难度的 mods](/wiki/Game_modifier#difficulty-increase)。
 
-### 要求
+然而值得注意的是，所有 Video Game Packs, Anime Packs, Internet! Pack 和 Rhythm Game Packs 自 [2008 年首次推出](https://osu.ppy.sh/community/forums/topics/1853) 以来发生了巨大变化。 有关处于旧版的曲包的列表，请参见[旧版曲包](/wiki/Medals/Legacy_beatmap_packs)。
```
Comment (kj415j45) [inline wiki/Medals/zh.md:11]:
```suggestion
然而值得注意的是，所有 Video Game Packs, Anime Packs, Internet! Pack 和 Rhythm Game Packs 自 [2008 年首次推出](https://osu.ppy.sh/community/forums/topics/1853) 以来已经发生了巨大变化。欲查看旧版曲包的列表，参见[旧版曲包](/wiki/Medals/Legacy_beatmap_packs)。
```

File: wiki/Medals/zh.md [hunk 5]
```diff
@@ -1,77 +1,59 @@
...
 # 奖章
 
-奖章，曾被称为 *成就*，是在用户个人资料页面中展示出来的一枚枚小型图案，可以体现玩家的游戏水准和技巧。
-当前有5种奖章：[曲包](#曲包)，[实力](#实力)，[毅力](#毅力)，[Mod 初体验](#Mod%20初体验)，和[隐藏](#隐藏)。
+**奖章**，曾被称为 *成就*，是在用户个人资料页面中展示出来的一枚枚小型图案，可以体现玩家的游戏水准和技巧。目前，在如下八个分类下有共计**247**枚奖章：[曲包](#曲包)，[实力](#实力)，[毅力](#毅力)，[Mod 入门](#Mod-入门)，[隐藏](#隐藏)，[谱面聚光灯](#beatmap-spotlights)，[季节聚光灯](#seasonal-spotlights)，和[谱面挑战曲包](#谱面挑战曲包)。
 
 ## 曲包
 
-**曲包**奖章的获得方法是： **完整体验特定曲包中的所有谱面** （每张至少选任意一难度）。您必须在 [osu!](/wiki/Game_mode/osu!) 模式中 pass 曲包中的每一张图，才能得到对应的奖章。
+**曲包奖章**在完成某些[主题曲包](https://osu.ppy.sh/beatmaps/packs?type=theme)或者[艺术家/专辑曲包](https://osu.ppy.sh/beatmaps/packs?type=artist)后获得。这些奖章通过完成曲包中每个谱面集的其中一个难度来获得。
```
Comment (kj415j45) [inline wiki/Medals/zh.md:7]:
```suggestion
**曲包奖章**在完成某些[主题曲包](https://osu.ppy.sh/beatmaps/packs?type=theme)或者[艺术家/专辑曲包](https://osu.ppy.sh/beatmaps/packs?type=artist)后获得。这些奖章通过完成曲包中每个谱面集的任意一个难度来获得。
```

File: wiki/Medals/zh.md (hunk 1 above)
Comment (Kotoki1337) [reply wiki/Medals/zh.md:3]: 括号内容可以考虑写在下文的原文后

File: wiki/Medals/zh.md [hunk 6]
```diff
@@ -1,77 +1,59 @@
...
+**Mod 入门**是在特定的[游戏模组（Mod）](/wiki/Game_modifier)下通过 [Qualified](/wiki/Beatmap/Category#qualified)，[Ranked](/wiki/Beatmap/Category#ranked)，或者 [Approved](/wiki/Beatmap/Category#approved) 谱面获得。这些奖章是为了激励新玩家尝试并体验各种各样的 Mod 而设立的。但是请注意，如果玩家同时使用多个 Mods。则玩家**不会**获得*任何* Mod 奖章。
 
----
+除非另有说明，否则这些奖章都可以在任何 [游戏模式](/wiki/Game_mode) 中解锁。
 
-排名奖章的获取方法是：在**任意**游戏模式中取得一定全球排名。
+## 隐藏
 
----
+**隐藏奖章**通过*神秘*的方式来获得，不会具体告诉玩家获得奖章的方法（因此得名）。使用特殊方法通过谱面。但是仍有一些提示给那些想要收集全奖章的人，如下所示：
```
Comment (kj415j45) [inline wiki/Medals/zh.md:31]:
```suggestion
**隐藏奖章**通过*神秘*的方式获得，不会告诉玩家如何获得奖章（因此得名）。取得方式从完成谱面并达成特殊要求，到使用一些非同寻常的 mod 组合都有。尽管如此，还是有一些提示给那些想要收集全奖章的人：
```

File: wiki/Medals/zh.md [hunk 7]
```diff
@@ -1,77 +1,59 @@
...
-谱面的星级将取整（如 5.88 星的图将会计为 5 星），并将给予玩家对应的奖章。
-使用难度增加或难度减少的 mods 来完成谱面或取得 S 评价也会计入奖章获取条件。
-注意，必须玩 Ranked 谱面才能获取奖章。
-以下展示的是8星的 *Pass* 和 *S* 奖章，实则从1星到8星都有对应的奖章。
+- 奖章的名字；
+- 奖章的简要说明；
+- 将鼠标放在奖牌卡片上的*谜语*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 -->（如果有）。
 
-## 毅力
+因为这些奖章意图就是隐藏的，解锁方式并未告知。但我们[欢迎玩家们一起探索如何解锁这些奖章](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals)。
```
Comment (kj415j45) [inline wiki/Medals/zh.md:37]:
```suggestion
由于这些奖章设计上就是隐藏成就，因此我们不会直接写明解锁方式。但我们[欢迎玩家们一起探索如何解锁这些奖章](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals)。
```

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: wiki/Medals/zh.md [hunk 8]
```diff
@@ -28,29 +28,29 @@
...
+由于这些奖章设计上就是隐藏成就，因此我们不会直接写明解锁方式。但我们[欢迎玩家们一起探索如何解锁这些奖章](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals)。
 
-## 谱面聚光灯
+## 第一期聚光灯
 
-**谱面聚光灯奖章**通过完成对应的[月度谱面聚光灯](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)曲包来获得。为了解锁此类奖章，玩家需要在四个模式中其中一个模式的谱面聚光灯排行榜竞争。允许使用[降低难度的 mods](/wiki/Game_modifier#difficulty-reduction) 和[提升难度的 mods](/wiki/Game_modifier#difficulty-increase)。
+**第一期聚光灯（月赛，原 charts）奖章**通过完成对应的[第一期聚光灯](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)曲包来获得。为了解锁此类奖章，玩家需要在四个模式中其中一个模式的第一期聚光灯排行榜竞争。允许使用[降低难度的 mods](/wiki/Game_modifier#difficulty-reduction) 和[提升难度的 mods](/wiki/Game_modifier#difficulty-increase)。
 
-虽然月度谱面聚光灯在 2018 年 4 月被[季节聚光灯](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)替代而不再发行，但时至今日，原有对应的奖章仍然可以获得。
+虽然第一期聚光灯在 2018 年 4 月被[第二期聚光灯](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)替代而不再发行，但时至今日，原有对应的奖章仍然可以获得。
```
Comment (kj415j45) [inline wiki/Medals/zh.md:43]:
```suggestion
虽然第一期聚光灯的奖章在 2018 年 4 月被[第二期聚光灯](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)替代而不再发行，但时至今日仍可以获得对应的奖章。
```

**Review by kj415j45 — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**